automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodConfigSpec.scala

```diff
@@ -13,7 +13,8 @@ import org.scalatest.{FreeSpec, Matchers, ParallelTestExecution}
 
 import scala.collection.immutable.ListMap
 
-class MethodConfigSpec extends FreeSpec with Matchers with WebBrowserSpec with WorkspaceFixtures with UserFixtures with MethodFixtures with BillingFixtures {
+class MethodConfigSpec extends FreeSpec with Matchers with WebBrowserSpec with WorkspaceFixtures with UserFixtures
+  with MethodFixtures with BillingFixtures with TestReporterFixture {
```

Comment:
`with TestReporterFixture` allows ScalaTest events write to test log (Test failed, Succeeded, etc.)